docs/proposals/quic-design.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Quic Design
+status: Alpha
+authors:
+    - "@Qianchenjie"
+approvers:
+creation-date: 2019-05-01
+last-updated: 2019-05-01
+---
+
+# Enhanced Cloud/Edge Communication
+
+## Abstract
+In order to enhance cloud and edge communication efficiency, we introduced the QUIC protocol for cloudhub and edgehub.
+I propose that cloudHub should support both websocket and QUIC protocol access at the same time. And the edgehub can choose one of the protocols to access to the cloudhub.
```

Comment:
dont use active voice. maybe say "it is suggested that"

Reply:
done